From ea89b4bebcfcb851d85d3a0086287b5056fe7a53 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 19 Oct 2018 12:12:48 +0200
Subject: [PATCH] Merge android-9.0.0_r12

commit f5e3553e300b206d5a3164999be31fc4613c1c68
Merge: e985808 b27063a
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Sun Jun 24 03:01:32 2018 +0000

    Snap for 4859140 from b27063a84b71567f114f2aea1dffd4f90ded8917 to pi-dr1-release

    Change-Id: I593e698dc5e607afed81bada83039fa96791e3f0

commit b27063a84b71567f114f2aea1dffd4f90ded8917
Author: Bill Yi <byi@google.com>
Date:   Thu Jun 21 13:00:44 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I19ad7c2967ced1d5e0a9e68ef6ade332e2887138

Change-Id: I384436d6dc092493693f8f30f0724f63655c22c0
---
 res/values-am/strings.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/values-am/strings.xml b/res/values-am/strings.xml
index 51af5c4..414bfc6 100644
--- a/res/values-am/strings.xml
+++ b/res/values-am/strings.xml
@@ -42,5 +42,5 @@
     <string name="stk_dialog_accept" msgid="8498901537508923727">"አዎ"</string>
     <string name="stk_dialog_reject" msgid="921848059485746796">"አይ"</string>
     <string name="no_sim_card_inserted" msgid="5316620398365235988">"የሲም መሥሪያ ስብስብን ለማስጀመር እባክዎ ሲም ያስገቡ።"</string>
-    <string name="stk_channel_name" msgid="8828587788561382056">"የተንቀሳቃሽ ስልክ አገልግሎት መልእክቶች"</string>
+    <string name="stk_channel_name" msgid="8828587788561382056">"የተንቀሳቃሽ ስልክ አገልግሎት መልዕክቶች"</string>
 </resources>
-- 
2.17.1

